data/json/recipes/practice/skills.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "id": "prac_computer_beg",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_SKILLS",
+    "name": "computer (beginner)",
+    "description": "Practice the basics of using a computer",
+    "skill_used": "computer",
+    "time": "30 m",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_cap": 2 },
+    "autolearn": [ [ "computer", 1 ] ],
+    "book_learn": [ [ "mag_computer", 0 ], [ "howto_computer", 0 ], [ "manual_computers", 0 ] ],
+    "tools": [ [ [ "laptop", 25 ], [ "eink_tablet_pc", 10 ], [ "smart_phone", 10 ] ] ],
+    "//": "Can be done in the dark due to lit screen",
+    "flags": [ "BLIND_EASY" ]
```

Comment:
Are computer practices disabled when the player is blindfolded or boomered?

Reply:
Hmm... I guess we don't have a flag that says you can do this in the dark, but not if you're blind. I'll remove these for now.